spring-integration-cassandra/src/main/java/org/springframework/integration/cassandra/outbound/CassandraMessageHandler.java

```diff
@@ -175,68 +197,74 @@ protected Object handleRequestMessage(Message<?> requestMessage) {
 
 		Object result = payload;
 
-		Type queryType = this.queryType;
+		OperationType queryType = this.queryType;
```

Comment:
Actually I called it like `Type` because it defines exactly the `CassandraMessageHandler` and its behavior.
Right, it is a operation anyway, but let's leave it as is for now.
The XML option may be called as `mode` (see `<int-jpa:outbound-channel-adapter>`) or `command` (see `<int-ftp:outbound-gateway>`).

I'm not sure yet, what have you changed here in the `CassandraMessageHandler`, but let's revert changes to avoid breaking changes, e.g. for Spring Cloud Data Flow: https://github.com/spring-cloud/spring-cloud-stream-modules/tree/master/cassandra-sink

If you'd like to change anything here, let's discuss that in the separate JIRAs!